tests/cli-rustup.rs

```diff
@@ -249,6 +249,25 @@ info: default toolchain set to 'nightly-{0}'
     });
 }
 
+#[test]
+fn default_override() {
+    setup(&|config| {
+        expect_ok(config, &["rustup", "update", "nightly", "--no-self-update"]);
+        expect_ok(config, &["rustup", "default", "stable"]);
+        expect_ok(config, &["rustup", "override", "set", "nightly"]);
+        expect_stderr_ok(
+            config,
+            &["rustup", "default", "stable"],
+            for_host!(
+                r"info: using existing install for 'stable-{0}'
+info: default toolchain set to 'stable-{0}'
+info: using override set by current environment
+info: nightly-{0} (directory override for"
```

Comment:
Sorry for the late review.

I'm a little concerned with the wording as to me it reads like the run-on sentence "(the) default toolchain (is) set to stable using override set by current environment" (which makes no sense) and then the line after just hangs there unexplained.

I was thinking:

```
info: using existing install for 'stable-{0}'
info: default toolchain set to 'stable-{0}'
info: note that the toolchain 'nightly-{0}' is currently in use (directory override for '{cwd}')
```

It might be a little long, so maybe best cut down to two lines.  But, what do you think?